.travis.yml

```diff
@@ -16,29 +16,57 @@ addons:
 
 env:
   global:
-  - DATABASE_URL='postgres://postgres@localhost/usaspending_api'
+  - USASPENDING_DB_USER=postgres
+  - DATABASE_URL=postgres://${USASPENDING_DB_USER}@localhost/usaspending_api
   - DJANGO_SETTINGS_MODULE='usaspending_api.settings'
   - ES_HOSTNAME='http://localhost:9200'
+  - BROKER_DB_HOST=localhost
```

Comment:
If the Broker DB is separated into env vars, should the USAspending config also be separated?